src/Components/Carousel/Carousel.css

```diff
@@ -23,43 +52,47 @@
   align-content: center;
 }
 
+
 .carousel {
+  position:static;
   grid-area: carousel;
-  width: 100px;
-  display: flex;
+  width:auto;
+  display:inline-flex;
   align-items: center;
   justify-content: center;
   gap: 8px;
+  z-index: 5;
 }
 .carousel-img {
   border: solid 2px #ccc0;
   display: flex;
   transition: all 750ms linear;
   width: 100%;
+  max-width: 150px;
```

Comment:
I'm not gonna comment it each time but for layout use the [relative](https://www.w3schools.com/cssref/css_units.php) (scroll down a bit)